superset-frontend/src/explore/components/PropertiesModal.tsx

```diff
@@ -236,14 +245,14 @@ function PropertiesModal({ slice, onHide, onSave }: InternalProps) {
               <label className="control-label" htmlFor="owners">
                 {t('Owners')}
               </label>
-              <SelectAsync
-                multi
+              <AsyncSelect
+                isMulti
                 name="owners"
                 value={owners || []}
                 loadOptions={loadOptions}
```

Comment:
Should the `loadOptions` function be returning an array of options instead of `{ options: [...] }`, like in the other `PropertiesModal`?

Reply:
You are right. Thanks for catching this! I forgot to test the properties modal in the Explore view.